analytics_plugin.py

```diff
@@ -50,23 +50,17 @@ def tasks_report_query(session, start_date, end_date) -> Any:
             Log.event.label("event"),
             func.count(Log.id).label("totalCount"),
         )
-        .select_from(Log)
-        .join(
-            TaskInstance,
-            and_(
-                Log.event.in_(
-                    [
-                        TaskInstanceState.SUCCESS,
-                        TaskInstanceState.FAILED,
-                    ]
-                ),
-                Log.dttm >= start_date,
-                Log.dttm <= end_date,
-                TaskInstance.operator != "DummyOperator",
-                TaskInstance.operator != "EmptyOperator",
-                Log.dag_id != "astronomer_monitoring_dag",
-                TaskInstance.task_id == Log.task_id,
+        .filter(
+            Log.dttm >= start_date,
+            Log.dttm <= end_date,
+            Log.dag_id != "astronomer_monitoring_dag",
+            Log.event.in_(
+                [
+                    TaskInstanceState.SUCCESS,
+                    TaskInstanceState.FAILED,
+                ]
```

Comment:
```suggestion
                ],
            Log.execution_date is not None,
```

or

```suggestion
                ],
            Log.extra is None,
```

Just experimenting a bit and noticed that marking TIs lead to audit log entries too. These are the obvious differences. Owner also varies, but (I suspect) if the TI owner and UI user name matches, you wouldn't be able to use that to differentiate.

Reply:
so exeuction_date not being set says it from the UI what does the extra is none do?

Reply:
I noticed 3 differences between the mark and task run.

![Screen Shot 2022-12-02 at 9 25 16 AM](https://user-images.githubusercontent.com/66968678/205339370-88ccc340-4261-4c07-8f34-2f18555292f6.png)

Reply:
ahh thx for that clarification! ill add those in

Reply:
@cmarteepants thoughts?

Reply:
Makes sense to me. Guessing that the extra field contains TI attributes that were used when marking the run with success.

Reply:
so we do want to remove  UI mark ones?

Reply:
Yes please